ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1038 @@
+
+== Description
+
+The EJB remote call contains a JTA transaction that is propagated over the EJB call.
+Further, the quickstart demonstrates the transaction recovery, which is run for both servers
+when a failure occurs.
+
+This quickstart contains two maven projects.
+The first maven project represents the caller side, and is intended to be deployed on the first server (`server1`).
+The second project represents the called side. This project is intended to be deployed
+to the other two servers (`server2` and `server2`). The two projects should not be deployed to the same server.
+
+.Maven projects in this quickstart
+[cols="40%,60%",options="headers"]
+|===
+|Project |Description
+
+|`client`
+|An application that you deploy to the first server, to `server1`.
+It provides REST endpoints that provides EJB remote calls to the `server application`
+residing on the other servers.
+Besides the remote EJB call the REST endpoint invocation processes data insertion to a database.
+As the REST invocation starts a transaction then the transaction enlists two participants
+(the database and the remote server called with EJB remote invocation).
+The transaction manager then uses the two-phase commit for committing the transaction.
+The quickstart examines failures and shows the transactional behaviour
+in such cases.
+
+|`server`
+|An application receives the remote EJB calls from the `client` application deployed at `server1`.
+This application is deployed to `server2` and `server3`.
+The EJB component receives the EJB remote call and depending on the scenario
+it resumes on the transaction propagated in the context of the remote EJB call.
+The call inserts data to the database.
+The applications also enlists a special XAResource to the transaction to demonstrate the quickstart.
+|===
+
+=== What to expect?
+
+The description is separated into two parts. The first one explains how to run the quickstart in a bare metal environment.
+The second one talks about deployment of the quickstart to Kubernetes and to OpenShift.
+
+The sequence of events to expected for bare metal installation:
+
+. Create three directories `server1`, `server2` and `server3` from installation directory of {productName}
+. Configure credentials on `server2` and `server2`
+. Configure remote outbound connection from `server1` to both servers `server2` and `server3`
+. Configure datasources, first on `server1` and then on both servers `server2` and `server3`
+. Start all 3 servers
+. Deploy the quickstart application to all servers.
+.. The deployed application at `server1` behaves as a client to both other servers.
+The `server1` offers several HTTP endpoints <<rest-endpoints>>. When the HTTP endpoint is invoked
+it starts an EJB call going to one of the servers `server2` or `server3`.
+. Call one of the HTTP endpoints at `server1`. Based on the scenario EJB remote calls
+hit the remote server and specific behaviour can be observed (see<<rest-endpoints>>).
```

Comment:
"Based on the scenario EJB remote calls hit the remote server and specific behaviour can be observed (see<<rest-endpoints>>)."

It is good practice to format a result such as this as a paragraph under the bullet.  Use a "+" on a line by itself to ensure the additional text is correctly indented.